lstchain/reco/r0_to_dl1.py

```diff
@@ -389,30 +399,80 @@ def r0_to_dl1(
                         # as the reference point. For the time being, the three TS
                         # are stored in the DL1 files for checking purposes.
 
-                        ucts_time = event.lst.tel[telescope_id].evt.ucts_timestamp * 1e-9  # secs
-
                         module_id = 82  # Get counters from the central Dragon module
 
                         if math.isnan(ucts_t0_dragon) and math.isnan(dragon_counter0) \
-                           and math.isnan(ucts_t0_tib) and math.isnan(tib_counter0):
+                                and math.isnan(ucts_t0_tib) and math.isnan(tib_counter0):
                             # Dragon/TIB timestamps not based on a valid absolute reference timestamp
-                            dragon_time = (event.lst.tel[telescope_id].svc.date +
-                                event.lst.tel[telescope_id].evt.pps_counter[module_id] +
-                                event.lst.tel[telescope_id].evt.tenMHz_counter[module_id] * 10**(-7))
 
-                            tib_time = (event.lst.tel[telescope_id].svc.date +
-                                event.lst.tel[telescope_id].evt.tib_pps_counter +
-                                event.lst.tel[telescope_id].evt.tib_tenMHz_counter * 10**(-7))
+                            dragon_time = (
+                                    event.lst.tel[telescope_id].svc.date +
+                                    event.lst.tel[telescope_id].evt.pps_counter[module_id] +
+                                    event.lst.tel[telescope_id].evt.tenMHz_counter[module_id] * 10 ** (-7)
+                            )
+
+                            tib_time = (
+                                    event.lst.tel[telescope_id].svc.date +
+                                    event.lst.tel[telescope_id].evt.tib_pps_counter +
+                                    event.lst.tel[telescope_id].evt.tib_tenMHz_counter * 10 ** (-7)
+                            )
```

Comment:
Since these timestamps are not valid timestamps I would suggest setting them as `nan` whenever they are not calculated using user-supplied first valid timestamps and counters.

Reply:
I'm ok with that solution, but this will imply a lot of `nan` and I'm not sure if anybody is using this information for anything, opinions?

Reply:
this is the only remaining thing of this PR, right?

Reply:
ok, there were more remaining things
https://github.com/cta-observatory/cta-lstchain/pull/362#discussion_r417389826
😄

Reply:
I would say keep it like it is. Or even calculate the timestamps using the initial ucts time if not provided by the user (even though they are not the ones from the first subrun). Don't know if the effort is worth it at this point.

Reply:
it is fine to leave it like this